server/models/interview.js

```diff
@@ -0,0 +1,37 @@
+let mongoose = require('mongoose');
+
+let Interview = new mongoose.Schema(
+	{
+		owner: {
+			type: mongoose.ObjectId,
+			required: true,
+		},
+		guest: {
+			type: mongoose.ObjectId,
+		},
+		date: {
+			type: Date,
+			required: true,
+		},
+		theme: {
+			type: String,
+		},
+		questions: {
+			type: [mongoose.ObjectId],
+		},
+		isFinished: {
+			type: Boolean,
+			default: false,
+		},
+		difficult: {
```

Comment:
```suggestion
		difficulty: {
```
I would change this name, because my intuition told me a variable called `difficult` would be a boolean.

Reply:
The field name has been updated as requested.